OpenRA.Game/WAngle.cs

```diff
@@ -31,6 +31,9 @@ public WAngle(int a)
 				Angle += 1024;
 		}
 
+		// Angle in the domain (-512, +512]
+		public int Angle2 { get { return Angle > 512 ? Angle - 1024 : Angle; } }
```

Comment:
This is no longer used, so we can drop 7d5d8b156b6503cedaffb24c3917e108847c047d.